irohad/ametsuchi/ametsuchi_stub.cpp

```diff
@@ -18,19 +18,25 @@
 #include <ametsuchi/ametsuchi_stub.hpp>
 #include <ametsuchi/mutable_storage_stub.hpp>
 #include <ametsuchi/temporary_wsv_stub.hpp>
+#include <iostream>
 
 namespace iroha {
   namespace ametsuchi {
 
     std::unique_ptr<TemporaryWsv> AmetsuchiStub::createTemporaryWsv() {
+      std::cout << "[\033[33mAmetsuchi\033[0m] creating temporary wsv" << std::endl;
```

Comment:
No, this is just for demonstration, we will change it after to logger. Can we assign to logger the color of the text ?